app/core/models.py

```diff
@@ -1,5 +1,203 @@
+        help_text='Can be retrieved from gh api with the following: curl -H \
+"Authorization: token [gh_PAT]" https://api.github.com/orgs/[org]',
+    )
+    github_primary_repo_id = models.CharField(
+        max_length=9,
+        blank=True,
+        help_text='Can be retrieved from gh api with the following: curl -H \
+"Authorization: token [gh_PAT]" \
+https://api.github.com/repos/[org]/[repo]',
+    )
+    github_primary_url = models.CharField(max_length=255, blank=True)
+    # current_status_id = models.ForeignKey("status", on_delete=models.PROTECT)
+    hide = models.BooleanField(default=True)
+    # location_id = models.ForeignKey("location", on_delete=models.PROTECT)
+    slack_url = models.URLField(blank=True)
+    google_drive_url = models.URLField(blank=True)
+    google_drive_id = models.CharField(max_length=255, blank=True)
+    hfla_website_url = models.URLField(blank=True)
+    # leads = models.ManyToManyField("lead")
+    # leadership_type_id = models.ForeignKey("leadership_type", on_delete=models.PROTECT)
+    image_logo = models.URLField(blank=True)
+    image_hero = models.URLField(blank=True)
+    image_icon = models.URLField(blank=True)
+    readme_url = models.URLField(blank=True)
+    wiki_url = models.URLField(blank=True)
+
+    def __str__(self):
+        return f"{self.name}"
+
+
+class RecurringEvent(AbstractBaseModel):
+    """
+    Recurring Events
+    """
+
+    name = models.CharField(max_length=255)
+    start_time = models.TimeField("Start", null=True, blank=True)
+    duration_in_min = models.IntegerField(null=True, blank=True)
+    video_conference_url = models.URLField(blank=True)
+    additional_info = models.TextField(blank=True)
+
+    project = models.ForeignKey(Project, on_delete=models.CASCADE)
+    # location_id = models.ForeignKey("Location", on_delete=models.DO_NOTHING)
+    # event_type_id = models.ForeignKey("EventType", on_delete=models.DO_NOTHING)
+    # brigade_id = models.ForeignKey("Brigade", on_delete=models.DO_NOTHING)
+    # day_of_week = models.ForeignKey("DayOfWeek", on_delete=models.DO_NOTHING)
+    # must_roles = models.ManyToManyField("Role")
+    # should_roles = models.ManyToManyField("Role")
+    # could_roles = models.ManyToManyField("Role")
+    # frequency_id = models.ForeignKey("Frequency", on_delete=models.DO_NOTHING)
+
+    def __str__(self):
+        return f"{self.name}"
+
+
+class SponsorPartner(AbstractBaseModel):
+    """
+    Dictionary of sponsors and partners
+    """
+
+    partner_name = models.CharField(max_length=255)
+    partner_logo = models.URLField(blank=True)
+    is_active = models.BooleanField(null=True)
+    url = models.URLField(blank=True)
+    is_sponsor = models.BooleanField(null=True)
+
+    # PK of this model is the ForeignKey for project_partner_xref
+
+    def __str__(self):
+        return f"{self.partner_name}"
+
+
+class Faq(AbstractBaseModel):
+    question = models.CharField(max_length=255, unique=True, null=True, blank=False)
```

Comment:
I missed this before, but [don't use null on text fields](https://docs.djangoproject.com/en/4.1/ref/models/fields/#:~:text=is%20False.-,Avoid%20using%20null,-on%20string%2Dbased).
You don't need to set blank=False since that's the default

Reply:
@fyliu ~ i get an error if i don't add null. i'll send you a screenshot in slack